### ISSUE TITLE
Drag events do not get handled by <Slider> trayProps

### ISSUE THREAD

**Issue by alexpreynolds** [express-labs/pure-react-carousel#220]
- `pure-react-carousel` version: master
- `react` version: v16.8.6
- `browser` used: Chrome v 78.0.3904.108 
- `node` version: v8.11.1

### Relevant code or config:

```js
function eventLogger(ev) {
  console.log(ev.type, ev.target);
}

return (
  <CarouselProvider ... dragEnabled={true}>
    <Slider trayProps={{
      onDrag: eventLogger,
      onDragEnd: eventLogger,
      onDragEnter: eventLogger,
      onDragExit: eventLogger,
      onDragLeave: eventLogger,
      onDragOver: eventLogger,
      onDragStart: eventLogger
    }}>
      <Slide>...</Slide>
      <Slide>...</Slide>
      <Slide>...</Slide>
    </Slider>
  </CarouselProvider>
)
```

### What you did:

Applied drag events on slides.

### What happened:

Nothing is reported by the `eventLogger` function.

### Reproduction:

See https://github.com/express-labs/pure-react-carousel/blob/fe25e75227b17101ae13a23e6421fbad6d36fe0b/src/App/examples/Example11/Example11.jsx for reproducible code.

### Problem description:

I would like to track drag events on slides, in order to get the current selected slide (which is used as state elsewhere in the web app).

Currently, I am only able to use the button `onClick` handlers to modify a "current-slide" state property.

### Suggested solution:

Enable drag events.

**Comment by bcarroll22** [comment]
Hey, thanks for reporting this!

Could you possibly help us debug why this is happening? It would also help get it released quicker if you could submit a PR to fix it if you find out why! Otherwise we will look into fixing it when we get the chance. 

Thanks again!

**Comment by vsolanogo** [comment]
So events for slide change through dragging not implemented yet? I need it like for the ButtonNext click...

**Comment by vsolanogo** [comment]
Are there any workarounds?

**Comment by victororlyk** [comment]
Yes it would be cool to have on drag event and have the ability to get the shown element index, so we may make for example video play when we end dragging and stop other videos wihtout big efforts..
Currently I am using onTouchEnd and do some checks there,  if index is another one then I do some stuff

**Comment by HoereeBauke** [comment]
any update on this @mrbinky3000 ?

**Comment by j-rp** [comment]
This repository looks to be unfortunately dead due to the [number of stale issues without response.](https://github.com/express-labs/pure-react-carousel/issues?q=is%3Aissue+label%3Astale). A shame, because I liked a lot of things about the component!

**Comment by tim-steele** [comment]
This has been answered https://github.com/express-labs/pure-react-carousel/issues/227

**Comment by thomasleduc** [comment]
Is this the recommended way to have a onClick on slide that doesn't trigger when dragging ?

**Comment by Conan1985** [comment]
Shouldn't this be closed as it has been answered bro #227?

**Comment by alexpreynolds** [comment]
I've moved on to other work, but I'll close this up for the time being.
